core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/symmetricDifferent/SymmetricDifferentOpDesc.scala

```diff
@@ -0,0 +1,34 @@
+package edu.uci.ics.texera.workflow.operators.symmetricDifferent
+
+import com.google.common.base.Preconditions
+import edu.uci.ics.amber.engine.operators.OpExecConfig
+import edu.uci.ics.texera.workflow.common.metadata.{
+  InputPort,
+  OperatorGroupConstants,
+  OperatorInfo,
+  OutputPort
+}
+import edu.uci.ics.texera.workflow.common.operators.{HashOpExecConfig, OperatorDescriptor}
+import edu.uci.ics.texera.workflow.common.tuple.schema.{OperatorSchemaInfo, Schema}
+
+class SymmetricDifferentOpDesc extends OperatorDescriptor {
+
+  override def operatorExecutor(operatorSchemaInfo: OperatorSchemaInfo): OpExecConfig = {
+    new HashOpExecConfig(operatorIdentifier, _ => new SymmetricDifferentOpExec())
+  }
+
+  override def operatorInfo: OperatorInfo =
+    OperatorInfo(
+      "SymmetricDifferent",
+      "Take the symmetric different of two inputs",
```

Comment:
please make the explanation more clear so that a naive user can understand what's the difference of `difference` operator and `SymmetricDifference` operator.

Reply:
Also please add an icon to each operator, the instructions are also in the guide, under `To add an image for this operator`

Reply:
Already modified.